R/fcast.R

```diff
@@ -105,18 +135,22 @@ aggregate_funs = function(funs, vals, sep="_", ...) {
   as.call(c(quote(list), unlist(ans)))
 }
 
-dcast.data.table = function(data, formula, fun.aggregate = NULL, sep = "_", ..., margins = NULL, subset = NULL, fill = NULL, drop = TRUE, value.var = guess(data), verbose = getOption("datatable.verbose")) {
+dcast.data.table = function(data, formula, fun.aggregate = NULL, sep = "_", ..., margins = NULL, subset = NULL, fill = NULL, drop = TRUE, value.var = guess(data), verbose = getOption("datatable.verbose"), value.var.in.dots = FALSE, value.var.in.LHSdots = value.var.in.dots, value.var.in.RHSdots = value.var.in.dots) {
   if (!is.data.table(data)) stopf("'data' must be a data.table.")
   drop = as.logical(rep(drop, length.out=2L))
   if (anyNA(drop)) stopf("'drop' must be logical TRUE/FALSE")
+  if (!value.var.in.dots %in% c(TRUE, FALSE))
```

Comment:
please use `isTRUEorFALSE` here and below instead of `%in%` which will not work for some odd inputs

Reply:
I can't find such function (`isTRUEorFALSE`). Do you mean to use `isTRUE(...) || isFALSE(...)`?

Reply:
Done now. Thanks!